pkg/controller/statefulset/stateful_set_control.go

```diff
@@ -17,6 +17,7 @@ limitations under the License.
 package statefulset
 
 import (
+	"k8s.io/kubernetes/pkg/kubelet/images"
```

Comment:
Import order: golang stdlib, 3rd party, k8s imports

Reply:
> Import order: golang stdlib, 3rd party, k8s imports

Thx~, I have fixed it.